addons/mrp/wizard/mrp_production_split.py

```diff
@@ -22,40 +23,50 @@ class MrpProductionSplit(models.TransientModel):
     product_qty = fields.Float(related='production_id.product_qty')
     product_uom_id = fields.Many2one(related='production_id.product_uom_id')
     production_capacity = fields.Float(related='production_id.production_capacity')
-    counter = fields.Integer(
-        "Split #", default=2, compute="_compute_counter",
-        store=True, readonly=False)
     production_detailed_vals_ids = fields.One2many(
         'mrp.production.split.line', 'mrp_production_split_id',
         'Split Details', compute="_compute_details", store=True, readonly=False)
     valid_details = fields.Boolean("Valid", compute="_compute_valid_details")
+    max_batch_size = fields.Float(
+        "Max Batch Size", default=2, compute="_compute_max_batch_size", readonly=False)
+    num_splits = fields.Integer(
+        "# Splits", compute="_compute_num_splits", readonly=True)
```

Comment:
```suggestion
    num_splits = fields.Integer("# Splits", compute="_compute_num_splits")
```

Reply:
@ticodoo As already discussed in log not , no need to make it editable. Thanks